benchmark_v2/parquet-fixed-strings_benchmark.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+
+import pytest
+
+import arkouda as ak
+
+
+@pytest.mark.skip_numpy(True)
+@pytest.mark.skip_correctness_only(True)
+@pytest.mark.benchmark(group="Parquet_Fixed_Strings")
+@pytest.mark.parametrize("scaling", [False, True])
+@pytest.mark.parametrize("fixed_size", [1, 8])
+@pytest.mark.parametrize("nfiles", [1])
+def bench_parquet_fixed_strings(benchmark, tmp_path, scaling, fixed_size, nfiles):
+    cfg = ak.get_config()
+    N = pytest.prob_size * cfg["numLocales"] if scaling else pytest.prob_size
+    base_path = Path(pytest.io_path) / f"parq_{'scaled' if scaling else 'flat'}_{nfiles}"
+    base_path.mkdir(parents=True, exist_ok=True)
```

Comment:
Should probably use the `tmp_path` fixture, according to the PR message?  Or adjust the PR message.